specs/unified_kernel_image.md

```diff
@@ -53,14 +53,15 @@ UKIs consist of the following resources:
 * Optionally, a microcode initrd in the `.ucode` section, to be handed to the kernel before any other initrd.
 * Optionally, a splash image to bring to screen before transitioning into the Linux kernel, in the `.splash` section.
 * Optionally, one or more compiled Device Trees, for systems which need it, each in its separate `.dtb` section. If multiple `.dtb` sections exist then one of them is selected according to an implementation-specific algorithm.
+* Optionally, one or more compiled Device Trees, for systems which need it, each in its separate `.dtbauto` section. If multiple `.dtbauto` sections exist then one of them is selected according to an implementation-specific algorithm.
 * Optionally, information describing kernel release information (i.e. `uname -r` output) in the `.uname` section. This is also useful for presentation of the UKI in the boot loader menu, and ordering it against other entries.
 * Optionally, a CSV file encoding the SBAT metadata for the image, in the `.sbat` section. The [SBAT format is defined by the Shim project](https://github.com/rhboot/shim/blob/main/SBAT.md), and used for UEFI revocation purposes.
 * Optionally, a JSON file encoding expected PCR 11 hash values seen from userspace once the UKI has booted up, along with signatures of these expected PCR 11 hash values, in the `.pcrsig` section. The signatures must also match the key pair described below.
 * Optionally, the public part of a public-private key pair in PEM format used to sign the expected PCR 11 value of the image, in the `.pcrpkey` section.
 
 Note that all of the sections defined above are singletons:
 they may appear at most once,
-except for the `.dtb` section which may appear multiple times.
+except for the `.dtb` and `.dtbauto` sections which may appear multiple times.
```

Comment:
i'd change this. .dtb should only be allowed once from now on. If people want multiple options they should use .dtbauto